crates/fluvio-spu-schema/src/server/smartmodule.rs

```diff
@@ -94,6 +94,15 @@ pub enum SmartModuleKind {
     },
     #[fluvio(min_version = ARRAY_MAP_WASM_API)]
     FilterMap,
+    #[fluvio(min_version = SMART_MODULE_API)]
```

Comment:
i think that we can put min_version and max_version in the same line like in https://github.com/infinyon/fluvio/blob/ed17bb1532cb493cd14d044d16ac18b15b9c4072/crates/fluvio-protocol/tests/version.rs#L8

Reply:
You are right. Thanks. Fixed.